addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -777,6 +777,50 @@ var ListFieldMany2One = FieldMany2One.extend({
     _renderReadonly: function () {
         this.$el.text(this.m2o_value);
     },
+
```

Comment:
To test: 
- readonly list : select 1 record > edit > ??
- click in unset m2o > click outside > ??

Reply:
Test result for second dot : focusing out an already empty m2o triggers a focusout, but not a field_changed (_setValue does nothing as the value is considered the same)